eppo_metrics_sync/dbt_model_parser.py

```diff
@@ -0,0 +1,128 @@
+from collections import Counter
+from itertools import chain
+
+
+class DbtModelParser():
+
+    def __init__(self, model, dbt_model_prefix):
+        self.model = model
+        self.eppo_entities = []
+        self.eppo_timestamp = []
+        self.eppo_facts = []
+        self.eppo_properties = []
+        self.dbt_model_prefix = dbt_model_prefix
+    
+    def parse_columns(self):
+
+        for column in self.model.get('columns'):
+
+            tags = column.get('tags')
+
+            if tags:
+                column_entities = [t for t in tags if 'eppo_entity:' in t]
+
+                if column_entities:
+                    for column_entity in column_entities:
```

Comment:
I think this block could have its own method as well—`self._parse_column_entities,` which would make the code less indented and better follow SRP.